docs/modules/ROOT/pages/integrating-with-existing-deployments-collapsed.adoc

```diff
@@ -0,0 +1,252 @@
+    The port number needs to be the management port - use `:18091` if you have TLS enabled.
+    There are alternative configuration options that do not require hard-coding the addresses - refer to the https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config[Prometheus documentation^] for more details.
+<.> Any labels defined here will be added to **all** metrics from this cluster - you may want to configure e.g., `environment`, `datacenter` etc.
+    The label `cluster` is required, and should match the name the cluster is configured to use in the Web Console.
+
+Add another `scrape_config` job for the previously-installed Cluster Monitor:
+
+.prometheus.yml
+[source, yaml]
+----
+...
+- job_name: couchbase-cluster-monitor
+      basic_auth:
+        username: admin  <.>
+        password: password <.>
+      metrics_path: /api/v1/_prometheus <.>
+      static_configs:
+          - targets: [10.145.212.110:7196] <.>
+...
+----
+
+<.> `username` should be the Username you configured for the Cluster Monitor.
+<.> `password` should be the Password you configured for the Cluster Monitor.
+<.> `metrics_path` is the path to the Prometheus server running on the Cluster Monitor. This should not be changed.
+<.> `targets` is a list of Cluster Monitor instances for Prometheus to retrieve metrics from. 
+  For now this should be a single server IP, listening on port `7196`.
+
+Restart Prometheus, and navigate to the Web UI. 
+If this was successful you should see your cluster listed on the page `/targets`.
+
+=== Prometheus: Alerting Rules
+
+CMOS ships with pre-made alert definitions which can alert you about certain events in your cluster. 
+To take advantage of these, first download the rule definitions to somewhere your Prometheus config can access them:
+
+[source, console]
+----
+wget -O prometheus-rules.yaml https://raw.githubusercontent.com/couchbaselabs/observability/main/microlith/prometheus/alerting/couchbase/couchbase-rules.yaml
+----
+
+Next, modify your Prometheus configuration file to include this directory. 
+For example, if you downloaded them to `/etc/prometheus/alerting/` then you would add:
+
+.prometheus.yml
+[source, yaml]
+----
+# Insert this at the top level of your prometheus.yml
+rule_files:
+  - /etc/prometheus/alerting/prometheus-rules.yaml
+----
+
+[NOTE]
+====
+If you have existing alert definitions, simply add the `prometheus-rules.yaml` file to the same directory, and specify `*.yaml` as the target for `rule_files`.
+====
+
+Restart Prometheus once more, and navigate to the Web UI. 
+You should be able to see the new rules listed on the `/rules` page.
+You may need to unhide inactive rules.
+
+=== Alertmanager
+
+We assume that you have already configured Prometheus to send alerts to Alertmanager, and that Alertmanager has appropriate receivers created and configured.
+
+Therefore, there is no additional configuration required.
+
+== Lets go
```

Comment:
:) (but could we use a different heading here?)

Reply:
such as?

Reply:
"Next Steps" or something like that? I know our docs aren't meant to be formal, but "let's go" seems a bit... too informal to me